### ISSUE TITLE
What is adj_train and how to create it?

### ISSUE THREAD

**Issue by mtlive** [GraphSAINT/GraphSAINT#1]
I wanted to try GraphSAINT on my data and could create specified files for my dataset except for adj_train.
At first I thought it's a graph with only train nodes but it contains all the node in adj_full. Then I tried to use adj_full as adj_train, it only works for node sampler and for other sampler I get this error:
`File "/content/GraphSAINT/graphsaint/minibatch.py", line 131, in set_sampler
    assert self.norm_loss_train[self.node_val].sum() + self.norm_loss_train[self.node_test].sum() == 0
AssertionError`
Is it the full graph without edges of test nodes? How can I create it from the full graph?

**Comment by GraphSAINT** [comment]
adj_train and adj_full both have shape |V| x |V|. However, adj_train[u,v] != 0 if and only if both u and v are training nodes. Essentially, in inductive learning, no information about test and validation sets (edge connection / features) is revealed during training. 

We insert the assertion to ensure the loss normalization is 0 for all val/test nodes. That's why when adj_train contains connections on non-training nodes, the program throws exception.

**Comment by mtlive** [comment]
I tried this for networkx graphs and it worked:
```
# Creating Train Graph, Assuming you have already created role.json from role
import copy
vtedges=[e for e in Gall.edges([*role['va'],*role['te']])] 
Gtrain=copy.deepcopy(Gall)
Gtrain.remove_edges_from(vtedges)
scipy.sparse.save_npz("adj_train",nx.adjacency_matrix(Gtrain))
```

**Comment by GraphSAINT** [comment]
Thanks for pointing out the issue and providing the snippet. We'll clarify this point in the README.

**Comment by omgmax** [comment]
@mtlive can you please clarrify what you mean by "Gall" in your snippet
